src/App/Documentation/components/InputGroup/index.js

```diff
@@ -1,12 +1,16 @@
 import React, { Component } from "react";
 
-import { ComponentPreview, DocToc } from "#";
+import { ComponentPreview, DocToc, Property, Attribute } from "#";
+import PrismCode from "react-prism";
 import InputGroupComponent from "@/InputGroup";
 
 const BasicExample = () => (
     <>
         <h2 id="basic-example">Basic example</h2>
-        <p>Basic input group...</p>
+        <p>
+            Add a span with class <Property value=".input-group-addon"/> to get a gray box on the left, right or both sides of an input element. The position of
```

Comment:
Should the words `span` and `input` be wrapped in `PrismCode` or something similar so it's obvious we're talking about the HTML element?

```suggestion
            Add a <PrismCode className="language-html">{"<span>"}</PrismCode> with class <Property value=".input-group-addon"/> to get a gray box on the left, right or both sides of an <PrismCode className="language-html">{"<input">}</PrismCode> element. The position of
```